### DISCUSSION TITLE
In the event that parsing of some fields fails, how to set a value to catch that failure

### DISCUSSION THREAD

**Discussion by yutakobayashidev** [open-circle/valibot#452] [Q&A] [answered]
In Zod, you can specify a value to be set when parsing of some values fails using .catch. Is there a similar solution in Valibot?

https://zod.dev/?id=catch

**Comment by fabian-hiller** [comment, marked as answer]
Yes, we call it `fallback`: https://valibot.dev/api/fallback/
